### PR TITLE
[Done] Translation of tools-testing.md ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

File: content/community/tools-testing.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 permalink: community/testing.html
 ---
 
-* **[Enzyme](https://github.com/airbnb/enzyme/):** a JavaScript Testing utility for React that makes it easier to assert, manipulate, and traverse your React Components' output.
-* **[Jest](https://facebook.github.io/jest/):** Delightful JavaScript testing used by Facebook to test all JavaScript code including React applications.
-* **[react-testing-library](https://github.com/kentcdodds/react-testing-library):** 🐐 Simple and complete React DOM testing utilities that encourage good testing practices.
-* **[React-unit](https://github.com/pzavolinsky/react-unit):** a lightweight unit test library for ReactJS with very few (js-only) dependencies.
-* **[Skin-deep](https://github.com/glenjamin/skin-deep):** Testing helpers for use with React's shallowRender test utils.
-* **[Unexpected-react](https://github.com/bruderstein/unexpected-react/):** Plugin for the [unexpected](https://unexpected.js.org/) assertion library that makes it easy to assert over your React Components and trigger events.
+* **[Enzyme](https://github.com/airbnb/enzyme/):** أداة اختبار جافا سكريبت لـ React التي تسهل تأكيد مخرجات مكوناته ومعالجتها والعبور عنها.
```
Comment (mohshbool) [inline content/community/tools-testing.md]:
```suggestion
* **[Enzyme](https://github.com/airbnb/enzyme/):** أداة اختبار جافا سكريبت لـ React التي تُسهّل تأكيد مخرجات مكوّناته ومعالجتها والعبور عنها.
```

File: content/community/tools-testing.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+* **[Jest](https://facebook.github.io/jest/):** اختبار جافا سكريبت رائع يستخدمه Facebook لاختبار جميع شفرات جافا سكريبت بما في ذلك تطبيقات React.
+* **[react-testing-library](https://github.com/kentcdodds/react-testing-library):** 🐐 أدوات اختبار بسيطة و كاملة لـ React DOM التي تشجع على الممارسات الجيدة للإختبار.
+* **[React-unit](https://github.com/pzavolinsky/react-unit):** مكتبة اختبار وحدات خفيفة لـ ReactJS مع عدد قليل جدا من التبعيات (فقط جافاسكريبت).
```
Comment (mohshbool) [inline content/community/tools-testing.md]: following the guidelines:
```suggestion
* **[React-unit](https://github.com/pzavolinsky/react-unit):** مكتبة اختبار وحدات خفيفة لـ ReactJS مع عدد قليل جدا من الإعتماديات (فقط جافاسكريبت).
```

File: content/community/tools-testing.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+* **[Skin-deep](https://github.com/glenjamin/skin-deep):** اختبار المساعدين للاستخدام مع أدوات اختبار التصيير السطحي (shallowRender) في React.
```
Comment (mohshbool) [inline content/community/tools-testing.md]:
```suggestion
* **[Skin-deep](https://github.com/glenjamin/skin-deep):** مُساعدين اختبار للاستخدام مع أدوات اختبار التصيّير السطحي (shallowRender) في React.
```

File: content/community/tools-testing.md (hunk 2 above)
Comment (3imed-jaberi) [reply content/community/tools-testing.md]: i think `التبعيات` better than `الإعتمادات` for --> `dependencies`  🤔 ..

Comment (mohshbool) [reply content/community/tools-testing.md]: but we already used that a lot, as I said. 😅

**Pull request merged**